### PR TITLE
Add a navigation RadioBox on Multiplayer screens

### PR DESCRIPTION
**Pull request by Naeferith**
The framed element is concerned

Radio Box
_- flyte's design -_

This is how the element is reacting on the visual test  : Radio Box

### CODE REVIEW

**Comment by Tom94** [comment]
IIRC the line below the labels should smoothly resize and move to the correct position rather than fading / appearing.

File: osu.Game/Screens/Multiplayer/MultiRadioBox.cs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+            foreach(Screens screen in Enum.GetValues(typeof(Screens)))
+            {
+                string label = screen.ToString();
+                if (label != "Lobby")
+                {
+                    multiplayerScreen.Add(new TabItem
+                    {
+                        Text = label,
+                        Active = false,
+                        Colour = new Color4(255, 255, 255, 179),
```
Comment (Tom94) [inline osu.Game/Screens/Multiplayer/MultiRadioBox.cs]: Hardcoding colors in multiple places is a bad idea.

File: osu.Game/Screens/Multiplayer/MultiRadioBox.cs [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                    Direction = FlowDirections.Horizontal,
+                    Anchor = Anchor.TopCentre,
+                    Origin = Anchor.TopCentre,
+                },
+            };
+
+            foreach(Screens screen in Enum.GetValues(typeof(Screens)))
+            {
+                string label = screen.ToString();
+                if (label != "Lobby")
```
Comment (Tom94) [inline osu.Game/Screens/Multiplayer/MultiRadioBox.cs]: String match and conditional construction of children seems weird. Why not just add all children as inactive, and then set the first one to active after the fact? In the future, the one to be active initially can then potentially be stored and changed from outside easily.

File: osu.Game/Screens/Multiplayer/MultiRadioBox.cs [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input;
+using osu.Game.Graphics.Sprites;
+using System;
+
+
+
+namespace osu.Game.Screens.Multiplayer
+{
+    public class MultiRadioBox : Container
```
Comment (Tom94) [inline osu.Game/Screens/Multiplayer/MultiRadioBox.cs]: Please make a general abstract class similar to the `DropDownMenu<T>` already existing in the framework, and put it into the framework. Something like `TabMenu<T>`. Then you can implement that abstract class here to give it your specific design and tabs.

**Comment by smoogipooo** [comment]
I believe this has been superseded by TabControl/OsuTabControl/FilterControl implementation.

**Pull request closed without merging**